.circleci/config.yml

```
@@ -227,6 +206,26 @@ jobs:
           name: Deploy beta
           command: ./scripts/deploy-beta-both
 
+  jest:
```

We can report feedback to devs earlier, and isolate it from other CI / Github output. Now the job runs top-level to the workflow, and one can see it in the Github PR status. If tests fail, blow up the PR early and visibly

---

```suggestion
  test-js:
```
why not keep the name?